catroid/src/org/catrobat/catroid/common/Constants.java

```diff
@@ -29,7 +29,7 @@
 	// Reflection in testcases needed
 	// http://stackoverflow.com/questions/1615163/modifying-final-fields-in-java?answertab=votes#tab-top
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.99f);
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(1.0f);
```

Comment:
it's only internal, but are we really *ready* for ``1.0``, or more for something like ``0.991``.  In other words: do we have a convention for this number, or are we happy as long as the value is increasing?